build/__variables.sh

```diff
@@ -44,6 +44,8 @@ declare -r ACR_SLIM_BUILD_IMAGE_REPO="$ACR_DEV_NAME/public/oryx/build-slim"
 declare -r ACR_RUNTIME_IMAGES_REPO="$ACR_PUBLIC_PREFIX"
 declare -r ACR_PACK_IMAGE_REPO="$ACR_PUBLIC_PREFIX/$PACK_IMAGE_NAME"
 
+declare -r BASE_IMAGES_REPO="$ACR_DEV_NAME/public/oryx/base"
```

Comment:
Reason for this PR is that we will just have one repo name to handle all our base images (some new ones are going to come) and to avoid explosion of repo names in MCR.

Reply:
But the prod acr needs to maintain the standard naming provided by mcr team. Make sure we follow that. I understand what you are trying here, but I'm afraid if we can do that in final/prod image. We can discuss his offline

Reply:
I read the example again, seems to be fine .. but I would still like to discuss other edge cases (may be)

Reply:
I already got this naming confirmed in MCR repo so I’m not clear what is the actual concern here?